### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by eshaanagarwal**
Changed CONTRIBUTING.md as requested  in #40

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -68,6 +68,9 @@ git remote add upstream https://github.com/duaimei/civic_hacking_hub.git
 2. Pull the latest changes from the main branch.
 
 ```bash
-git pull upstream main
+$ git fetch --all
+$ git checkout dev
+$ git pull
+$ git checkout -b
```
Comment (amycommits) [inline CONTRIBUTING.md]: would you mind putting something like:
$ git checkout -b \<your-branch-name\>

Comment (amycommits) [reply CONTRIBUTING.md]: The trick is you need to add a \\ in front of the \<

**Review by amycommits — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (eshaanagarwal) [reply CONTRIBUTING.md]: Sure! I will make the changes.

**Comment by eshaanagarwal** [comment]
@duaimei I have made the required changes! Pls review.

**Review by amycommits — APPROVED** [review]

**Comment by amycommits** [comment]
Thank you, sorry that took so long!

**Pull request merged**